scripts/create-json-for-web.sh

```
@@ -0,0 +1,55 @@
+if ! command -v poetry &> /dev/null; then
+    echo "poetry could not be found"
+    exit 1
+fi
+
+# cd to parent directory of this script
+cd $(dirname $(dirname $(realpath $0)))
+
+work_dir=
+while getopts "i:o:" opt; do
+    case $opt in
+        w) work_dir=$OPTARG ;;
```

Shellcheck (0.10.0)

[warning] This case is not specified by getopts.

(SC2214)